packages/core/src/awsService/eventBridgeScheduler/commands/createSchedule.ts

```diff
@@ -0,0 +1,168 @@
+import { showQuickPick, showInputBox } from '../../../shared/ui/pickerPrompter'
+import { createQuickStartUrl } from '../../../shared/utilities/workspaceUtils'
+
+/**
+ * Command to create a new EventBridge Scheduler schedule
+ * 
+ * This command guides users through creating schedules for automated task execution.
+ * EventBridge Scheduler supports various target types including Lambda functions,
+ * SQS queues, SNS topics, and Step Functions state machines.
+ * 
+ * Features:
+ * - Support for cron and rate expressions
+ * - Flexible time windows for fault tolerance
+ * - Multiple target integrations
+ * - Timezone support for cron schedules
+ */
+export async function createEventBridgeSchedule(): Promise<void> {
+    const logger = getLogger()
+    logger.info('Starting EventBridge Scheduler create schedule workflow')
+
+    try {
+        const schedulerService = new EventBridgeSchedulerService()
+        
+        // Get schedule name
+        const scheduleName = await showInputBox({
+            title: 'Schedule Name',
+            placeholder: 'my-daily-backup-schedule',
+            validateInput: (input) => {
+                if (!input || input.trim().length === 0) {
+                    return 'Schedule name is required'
+                }
+                if (input.length > 64) {
+                    return 'Schedule name must be 64 characters or fewer'
+                }
+                if (!/^[a-zA-Z0-9\-_]+$/.test(input)) {
+                    return 'Schedule name can only contain letters, numbers, hyphens, and underscores'
+                }
+                return undefined
+            }
+        })
+
+        if (!scheduleName) {
+            return
+        }
+
+        // Get schedule type
+        const scheduleType = await showQuickPick([
+            { label: 'Rate-based', detail: 'Run at regular intervals (every X minutes/hours/days)' },
+            { label: 'Cron-based', detail: 'Run based on cron expression (specific times/dates)' },
+            { label: 'One-time', detail: 'Run once at a specific date and time' }
+        ], {
+            title: 'Schedule Type',
+            ignoreFocusOut: true
+        })
+
+        if (!scheduleType) {
+            return
+        }
+
+        // Get schedule expression based on type
+        let scheduleExpression: string
+        switch (scheduleType.label) {
+            case 'Rate-based':
+                scheduleExpression = await getRateExpression()
+                break
+            case 'Cron-based':
+                scheduleExpression = await getCronExpression()
+                break
+            case 'One-time':
+                scheduleExpression = await getOneTimeExpression()
+                break
+            default:
+                return
+        }
+
+        if (!scheduleExpression) {
+            return
+        }
+
+        // Get target type
+        const targetType = await showQuickPick([
```

Comment:
Consider adding error handling for the case when `schedulerService.createSchedule()` is implemented. Currently, the error is caught but not handled specifically.
```suggestion
try {
    await schedulerService.createSchedule(scheduleName, scheduleExpression, target);
    await vscode.window.showInformationMessage(`Schedule "${scheduleName}" created successfully.`);
} catch (error) {
    logger.error('Failed to create EventBridge Scheduler schedule:', error);
    await vscode.window.showErrorMessage(`Failed to create schedule: ${error}`);
}
```